src/azul/indexer/__init__.py

```diff
@@ -403,6 +405,26 @@ def drs_path(self, manifest_entry: JSON) -> Optional[str]:
         """
         raise NotImplementedError
 
+    def _reject_if_contains(self, match: str, contents: Any):
```

Comment:
```suggestion
    def _reject_if_contains(self, needle: str, haystack: Any):
```

The word *match* is typically used to signify something that has already been found, not something to look for.